build.conf.arm

```diff
@@ -0,0 +1 @@
+COMPILED_KERNEL_URL=https://github.com/rancher/os-kernel/releases/download/HypriotOS-4.1.12-v7+/linux-4.1.12-hypriotos-v7.-arm.tar.gz
```

Comment:
This build actually runs on RPi, so I guess, yes it does. What libs do you mean?

I'm going to try Raspbian kernel, just for funzies, as well. Hypriot was the easiest to start with as docker is known to work on it.